### DISCUSSION TITLE
DFTB+ MD gets stuck

### DISCUSSION THREAD

**Discussion by achyuttt** [dftbplus/dftbplus#1519] [General]
Hello everyone,

I was running DFTB+ MD for a system of 92 atoms in it using mpirun. The calculation was so fast until 900 iteration that it was completed at around 23 minutes with a SCC Tolerance of 10E-6 however after 905th iteration, the calculation runs but it doesn't print any output in the output files.

What could be the reason?

**Comment by aradi** [comment]
It could be a problem with your MPI- or Scalapack- framework, or a bug in DFTB+ (some kind of race condition leading to a lock). It is hard to tell. Also, make sure to check error messages or anything alike. It could also be, that DFTB+ actually stoped with an error, but could not shut down the MPI-framework correctly.

**Reply by achyuttt** [reply]
I checked out all the log files but it indicates no errors there. I used to face similar kind of problems but previously it used to be due to sudden rise in the temperature of the system and there used to be NaN errors. But now it stops printing the forces half way but in the cluster the simulation keeps running. I have installed it in a clean environment using conda. 

I have no idea on what could it be.
